python-backend/app/api/mines/mine/models/mine_type.py

```diff
@@ -47,7 +41,8 @@ def find_by_guid(cls, _id):
         uuid.UUID(_id, version=4)
         return cls.query.filter_by(mine_type_guid=_id).first()
 
-    @classmethod
-    def expire_record(cls, record):
-        record.active_ind = False
-        record.save()
+    def expire_record(self):
```

Comment:
"expire record" seems awkward, why not call it "inactivate" which describes the logic - setting active = false.